go.mod

```diff
@@ -3,8 +3,8 @@ module github.com/taskcluster/mozilla-history
 go 1.13
 
 require (
-	github.com/google/uuid v1.1.1 // indirect
 	github.com/taskcluster/httpbackoff/v3 v3.1.0
 	github.com/taskcluster/slugid-go v1.1.0
 	github.com/taskcluster/taskcluster/v30 v30.0.2
```

Comment:
does `go mod tidy` remove taskcluster v30 here?

Reply:
hmm. nope, it didn't. Will try to drop manually then

Reply:
aah, it's being referenced from the `main.go` on the top level.. will update that one as well